examples/TakeImageWithStandardScript.ipynb

```diff
@@ -0,0 +1,352 @@
+{
```

Comment:
Looks OK to me. I'm leery of examples due to bitrot, as you know. I don't much care if they are in the package or here -- either way I am fairly sure (based on my DM experience) that they will quickly become useless.

Another option is to write examples as unit tests, so they get exercised. They don't actually have to *look* like unit tests -- with asserts and all that -- they just have to run the code (in such a way that if something bit rots that an exception will make the test fail -- which you pretty much get for free. You *might* even be able to make scons or pytest run notebooks, but I don't know how to do it.